web/client/epics/layers.js

```diff
@@ -86,7 +89,26 @@ const refresh = action$ =>
             .mergeAll();
         });
 
-
+/**
+ * Update dimension to all layers that have that dimension set, or for the layers indicated in the action.
+ * @memberof epics.layers
+ * @param {external:Observable} action$ manages `UPDATE_LAYERS_DIMENSION`
+ * @return {external:Observable}
+ */
+const updateDimension = (action$, {getState = () => {}} = {}) =>
```

Comment:
What is the side effect here?

Reply:
What do you mean? Triggers a changeLayerParams for the layers passed as property or all the layer that have the dimension, if layers array is empty.
Notice that I have extended CHANGE_LAYER_PARAMS (PROPERTIES) to accept 1 string or an array of string for layers to change, to make multiple updates easier.